### ISSUE TITLE
Accessible keyboard shortcut help not displaying in Electron

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#10947]
### System details

    RStudio Edition : Desktop (Electron)
    RStudio Version : 2022.06.0-daily+284
    OS Version      : macOS 12.3.1 (also reproduced on Windows-11 with same build)
    R Version       : any

### Steps to reproduce the problem

- Start the IDE and make sure screen reader support is disabled (Help / Accessibility / Screen Reader Support)
- Help / Keyboard Shortcuts Help
- A semi-transparent popup window displays showing keyboard shortcut help (expected)
- Turn on screen reader support and restart
- Help / Keybaord Shortcuts Help

### Describe the problem in detail

- Nothing happens

### Describe the behavior you expected

- A web page should be loaded in the web browser showing keyboard shortcut help; this page is screen-reader accessible
- The page does show when using the same Qt-based build
- You can also reach this page when screen-reader support is disabled by clicking the "See all shortcuts..." link in the upper-right corner of the non-accessible keyboard shortcut popup (this also doesn't work in Electron)

- [x] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [x] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [x] If I am reporting an RStudio crash, I have included a [diagnostics report](https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.
